### DISCUSSION TITLE
Add option for skipping step 6 on group stage

### DISCUSSION THREAD

**Discussion by bondrewd** [pkolaczk/fclones#188] [Ideas]
Hi! First of all, let me thank you for this fantastic tool. It has helped me a lot with my sys admin tasks.

I want to ask if the possibility of letting the user opt out from hashing the whole content of the files has been considered. Of course, this will likely increase the possibility of false positives and negatives, but it is convenient for quick and dirty jobs. I found it helpful when working with systems with a lot of files (in the order of ~1 PB).

**Comment by pkolaczk** [comment]
This has been added in 0.31 by #202

**Reply by bondrewd** [reply]
Nice! Thank you very much!!!
